components/event-bus/poc/README.md

```diff
@@ -0,0 +1,749 @@
+## [POC] Run EventBus on top of Knative NatssChannel CRD
+
+### Export GKE cluster properties
+```bash
+export CLUSTER_NAME="sayan-event-bus" \
+export GCP_PROJECT="kyma-project" \
+export GCP_ZONE="europe-west1-b" \
+export CLUSTER_VERSION="1.12.7"
+```
+
+### Create a GKE cluster
+```bash
+gcloud container --project "$GCP_PROJECT" clusters create "$CLUSTER_NAME" \
+--addons=HorizontalPodAutoscaling,HttpLoadBalancing \
+--machine-type "n1-standard-4" \
+--cluster-version "$CLUSTER_VERSION" --zone "$GCP_ZONE" \
+--enable-stackdriver-kubernetes --enable-ip-alias \
+--enable-autoscaling --min-nodes=1 --max-nodes=10 \
+--enable-autorepair \
+--scopes cloud-platform
+```
+
+### Grant cluster-admin permissions
+```bash
+kubectl create clusterrolebinding cluster-admin-binding \
+  --clusterrole=cluster-admin \
+  --user=$(gcloud config get-value core/account)
+```
+
+### Install Istio
+```bash
+kubectl apply --filename https://raw.githubusercontent.com/knative/serving/v0.8.0/third_party/istio-1.1.7/istio-crds.yaml
+
+curl -L https://raw.githubusercontent.com/knative/serving/v0.8.0/third_party/istio-1.1.7/istio.yaml \
+  | kubectl apply -f -
+
+# label the default namespace with istio-injection=enabled.
+kubectl label namespace default istio-injection=enabled
+
+# watch pods until is shows STATUS of Running or Completed
+kubectl get pods --namespace istio-system -w
+```
+
+### Install Knative
+```bash
+kubectl apply --selector knative.dev/crd-install=true \
+   --filename https://github.com/knative/serving/releases/download/v0.8.0/serving.yaml \
+   --filename https://github.com/knative/eventing/releases/download/v0.8.0/release.yaml
+
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.8.0/serving.yaml --selector networking.knative.dev/certificate-provider!=cert-manager \
+   --filename https://github.com/knative/eventing/releases/download/v0.8.0/release.yaml
+```
+
+### Install NATSS server
+```bash
+kubectl create namespace natss; \
+kubectl apply -n natss -f https://raw.githubusercontent.com/knative/eventing/v0.8.0/contrib/natss/config/broker/natss.yaml
+```
+
+### Install NATSS controller and dispatcher
+```bash
+# export your docker registry and use ko to install NATSS controller and dispatcher
+kubectl apply -n knative-eventing -f https://github.com/knative/eventing/releases/download/v0.8.0/natss.yaml
+```
+
+### Remove default configmap for CCP
+
+```bash
+kubectl delete cm default-channel-webhook -n knative-eventing
+```
+
+### Add a default configmap for Channel
+```bash
+cat << EOF | kubectl apply -f -
```

Comment:
The config map already exists with knative 0.8. Either edit it or delete it beforehand